src/AltinnCore/Designer/Controllers/CompileController.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Threading.Tasks;
+using AltinnCore.Common.Helpers;
+using AltinnCore.Common.Models;
+using AltinnCore.Common.Services.Interfaces;
+using AltinnCore.ServiceLibrary.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace Designer.Controllers
+{
+    /// <summary>
+    /// Controller for compile API endpoint
+    /// </summary>
+    [ApiController]
+    [Authorize]
+    [Route("designer/api/v1/[controller]")]
```

Comment:
Maybe we need to update configMap on the cluster for this route to work?

Reply:
That is very likely. Mind looking at this together?